app/controllers/course/assessment/submissions_controller.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 class Course::Assessment::SubmissionsController < Course::Assessment::Controller
-  include Course::Assessment::SubmissionControllerWorksheetConcern
-  include Course::Assessment::SubmissionControllerGuidedConcern
+  include Course::Assessment::SubmissionControllerServiceConcern
+  delegate_to_service(:update)
```

Comment:
I would put this after the before/after hooks since that's where I expect the actions to be at.